hawkbit-extension-multi-user/README.md

```diff
@@ -0,0 +1,25 @@
+# hawkBit multi user extension
+
+This extension allows you to statically configure multiple users through the application properties.
+When the extension is enabled but no users are configured, it will fall back to the standard security.user configuration.
+
+## Example configuration
+
+A list of permissions with description is [available from the hawkBit repository](https://github.com/eclipse/hawkbit/blob/master/hawkbit-security-core/src/main/java/org/eclipse/hawkbit/im/authentication/SpPermission.java).
+Additionally, if a single permission with the name *ALL* is configured, that user will be granted all available permissions.
+
+    multiuser.user[0].username=admin
```

Comment:
Since it is a list of users I'd change it to `users` (like it is for `permissions`). Getter/Setter for `users` will look better as well within property class 
```suggestion
    multiuser.users[0].username=admin
```